homeassistant/components/nuki/strings.json

```diff
@@ -1,4 +1,10 @@
 {
+  "issues": {
+    "local_https_url": {
+      "title": "Local Network address configured with an HTTPS URL",
+      "description": "You Local Network address for Home Assistant is configured with an HTTPS URL. This needs to be an HTTP URL for the Nuki bridge to be able to push events to Home Assistant."
```

Comment:
```suggestion
      "description": "Your local network address for Home Assistant is configured with an HTTPS URL. This needs to be an HTTP URL for the Nuki bridge to be able to push events to Home Assistant."
```

Reply:
Additionally, this needs to explain to the user what they need to do and how they can fix this issue.

Right now, this is more of a notification, without any clear path for the user on how to solve the problem and how to dismiss the issue raised by this integration.

Reply:
I changed and added a sentence.